### PR TITLE
GCG on windows

### PR DESCRIPTION
**Pull request by jurgen-lentz**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jurgen-lentz** [comment]
@AntoinePrv Can you maybe look at this? Papilo does fail, gcg compiles successfully:
```
FAILED: bin/papilo.exe 
C:\Windows\system32\cmd.exe /C "cd . && %BUILD_PREFIX%\Library\bin\cmake.exe -E vs_link_exe --intdir=papilo\binaries\CMakeFiles\papilo-executable.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1429~1.301\bin\HostX64\x64\link.exe /nologo papilo\binaries\CMakeFiles\papilo-executable.dir\__\src\papilo.cpp.obj  /out:bin\papilo.exe /implib:papilo\binaries\papilo.lib /pdb:bin\papilo.pdb /version:0.0 /machine:x64 /INCREMENTAL:NO /subsystem:console  papilo\libpapilo-core.lib  lib\libsoplex.lib  lib\libscip.lib  %PREFIX%\Library\lib\boost_iostreams.lib  %PREFIX%\Library\lib\boost_serialization.lib  %PREFIX%\Library\lib\boost_program_options.lib  %PREFIX%\Library\lib\tbb12.lib  -lpthreads  %PREFIX%\Library\lib\z.lib  %PREFIX%\Library\lib\gmp.lib  %PREFIX%\Library\lib\mpfr.lib  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."
LINK: command "C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1429~1.301\bin\HostX64\x64\link.exe /nologo papilo\binaries\CMakeFiles\papilo-executable.dir\__\src\papilo.cpp.obj /out:bin\papilo.exe /implib:papilo\binaries\papilo.lib /pdb:bin\papilo.pdb /version:0.0 /machine:x64 /INCREMENTAL:NO /subsystem:console papilo\libpapilo-core.lib lib\libsoplex.lib lib\libscip.lib %PREFIX%\Library\lib\boost_iostreams.lib %PREFIX%\Library\lib\boost_serialization.lib %PREFIX%\Library\lib\boost_program_options.lib %PREFIX%\Library\lib\tbb12.lib -lpthreads %PREFIX%\Library\lib\z.lib %PREFIX%\Library\lib\gmp.lib %PREFIX%\Library\lib\mpfr.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST:EMBED,ID=1" failed (exit code 1104) with the following output:
LINK : warning LNK4044: unrecognized option '/lpthreads'; ignored
   Creating library papilo\binaries\papilo.lib and object papilo\binaries\papilo.exp

LINK : fatal error LNK1104: cannot open file 'papilo\binaries\papilo.exp'

[1346/1346] Linking CXX shared library papilo\binaries\papilo.dll
```

File: recipe/meta.yaml [hunk 1]
```diff
@@ -170,32 +169,37 @@ outputs:
...
-      - "lib/libgcg*"
-      - "lib/cmake/gcg"
-      - "bin/gcg"
-      - "include/gcg"
+      - "{{ install_prefix }}/lib/libgcg*"
+      - "{{ install_prefix }}/lib/cmake/gcg"
+      - "{{ install_prefix }}/bin/gcg*"
+      - "{{ install_prefix }}/bin/libgcg*"  # [win]
+      - "{{ install_prefix }}/include/gcg"
+      - "{{ install_prefix }}/include/graph"
```
Comment (AntoinePrv) [inline recipe/meta.yaml:193]: Is `graph` part of the `gcg` headers?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -106,7 +106,6 @@ outputs:
       - "{{ install_prefix }}/include/blockmemshell/"
       - "{{ install_prefix }}/include/dijkstra/"
       - "{{ install_prefix }}/include/lpi/"
-      - "{{ install_prefix }}/include/nlpi/"
```
Comment (AntoinePrv) [inline recipe/meta.yaml:109]: Why do we remove this?

**Review by AntoinePrv — COMMENTED** [review]
Thanks for looking into it. Unfortunately I don't know  enough about GCG and Windows to help.
I just have a few questions and this should be good to go.

File: recipe/meta.yaml (hunk 1 above)
Comment (jurgen-lentz) [reply recipe/meta.yaml:193]: Yes it is.

File: recipe/meta.yaml (hunk 2 above)
Comment (jurgen-lentz) [reply recipe/meta.yaml:109]: There is no nlpi folder.

**Comment by jurgen-lentz** [comment]
> Thanks for looking into it. Unfortunately I don't know enough about GCG and Windows to help. I just have a few questions and this should be good to go.

Can you please merge it?

**Pull request merged**